server/parameters/util/OptionParser.java

```diff
@@ -58,6 +58,11 @@ FunctionalIterator<Option> matchingOptions(Set<Option> options) {
         return iterate(options).filter(opt -> opt.name().equals(name()));
     }
 
+    @Override
+    public java.lang.String toString() {
```

Comment:
if we end up throwing an exception with an `OptionParser` in it - we should have a useful `toString()`, which should just be the help entry for that parser